src/lib/expansion/_expansion-theme.scss

```diff
@@ -56,4 +56,8 @@
   .mat-expansion-panel-content {
     @include mat-typography-level-to-styles($config, body-1);
   }
+
+  .mat-expansion-panel-header-title {
```

Comment:
font-weight styling should go into `@mixin mat-expansion-panel-typography`.

Reply:
This is inside the typography mixin already.